webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -0,0 +1,205 @@
+                </p>
+            );
+        }
+
+        if (!visible) {
+            return null;
+        }
+
+        let submitError = null;
+        if (error) {
+            submitError = (
+                <p className='help-text error-text'>
+                    <span>{error}</span>
+                </p>
+            );
+        }
+
+        const component = (
+            <div>
+                <GithubRepoSelector
+                    onChange={this.handleRepoValueChange}
+                    value={this.state.repoValue}
+                    required={true}
+                    theme={theme}
+                    addValidate={this.validator.addComponent}
+                    removeValidate={this.validator.removeComponent}
+                />
+                <Input
+                    id={'title'}
+                    label='Title for the GitHub Issue'
+                    type='input'
+                    required={true}
+                    disabled={false}
+                    maxLength={65}
+                    value={this.state.issueTitle}
+                    onChange={this.handleIssueTitleChange}
+                />
+                {issueTitleValidationError}
+                <Input
+                    label='Description for the GitHub Issue'
+                    type='textarea'
+                    value={this.props.post.message}
+                    disabled={true}
+                />
+            </div>
+        );
+
+        return (
+            <Modal
+                dialogClassName='modal--scroll'
+                show={visible}
+                onHide={this.handleClose}
+                onExited={this.handleClose}
+                bsSize='large'
+                backdrop='static'
+            >
+                <Modal.Header closeButton={true}>
+                    <Modal.Title>
+                        {'Create GitHub Issue'}
+                    </Modal.Title>
+                </Modal.Header>
+                <form
+                    role='form'
+                    onSubmit={this.handleCreate}
+                >
+                    <Modal.Body
+                        style={style.modal}
+                        ref='modalBody'
+                    >
+                        {component}
+                    </Modal.Body>
+                    <Modal.Footer>
+                        {submitError}
+                        <FormButton
+                            type='button'
+                            btnClass='btn-link'
+                            defaultMessage='Cancel'
+                            onClick={this.handleClose}
+                        />
+                        <FormButton
+                            type='submit'
+                            btnClass='btn btn-primary'
+                            saving={submitting}
+                            defaultMessage='Submit'
+                            savingMessage='Submitting'
+                        >
+                            {'Submit'}
```

Comment:
The icon doesn't render for me in chromium 80
![Screenshot from 2020-04-16 14-16-10](https://user-images.githubusercontent.com/16541325/79455270-e9381e80-7fec-11ea-964c-e3d01fcd3892.png)

Reply:
I see.  I committed a change to the form_button css classes and believe this is now fixed.  @asaadmahmood, does this seem correct with this new commit?  https://github.com/mattermost/mattermost-plugin-github/pull/239/commits/4e7e89e54e53d926c66f7aecfbc280b3a29cc089

Reply:
Works now :+1: